### ISSUE TITLE
Any way to use Xbox One controller with a WMR headset?

### ISSUE THREAD

**Issue by LasagnaSmoothie** [Nibre/MotherVR#171]
Hello,

I'm having an issue using an Xbox One controller with the MotherVR mod and a Samsung Odyssey+ WMR headset.  The WMR controllers work acceptably with the mod, however, I'd really prefer to play the game with a standard Xbox One controller.  Unfortunately, when I use my Xbox controller I encounter one of two issues:

-The title screen menu system often goes crazy as though a button on the controller is being clicked over and over again in rapid succession.
-If I don't encounter the crazy title screen issues, when I click to start a scenario, the game inevitably crashes before it begins loading the scenario.

Any help/suggestions would be hugely appreciated.  Thanks.  (And a huge thanks to the fantastic work the creator of this mod has done.)

**Comment by thedooker** [comment]
I would get the rapid button press whenever I started the game without the motion controllers turned on. What I do is turn on my gamepad and **both** motion controllers(1 doesn't seem to stop it) then boot the game. I'll press a button on the gamepad to make sure the game recognizes it, then I turn off the motion controllers and I'm usually good. 

Played the past 2 weeks with my PS4 controller doing this and haven't had any issues. Which is strange because I used to have the same issue of not being able to crouch or the infinite button press as others did with a Xbox controller until recently.
